### ISSUE TITLE
[Input]:[Custom Suggestions][A11y] Additional text value is not reading out in screen reader NVDA/JAWS

### ISSUE THREAD

**Issue by VidhiNPatel** [SAP/ui5-webcomponents#12170]
### Bug Description

When we use the `<ui5-suggestion-item-custom>`, the additional text value (end of the list item) is not read out in screen reader NVDA/JAWS

### Affected Component

<ui5-suggestion-item-custom>

### Expected Behaviour

The additional text value (end of the list item) should read via screen reader NVDA/JAWS

### Steps to Reproduce

1. Go to https://sap.github.io/ui5-webcomponents/components/Input/#custom-suggestions
2. Turn on the Screen reader
3.  Open the suggestion popover and verify that the additional text value EU (Green Color) is not read out via screen reader

### Priority

High

### UI5 Web Components Version

2.13

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-rl ,

Could you take a look at this issue with SuggestionItemCustom?

Regards,
Todor

**Comment by StefanDimitrov04** [comment]
Hello colleagues,

```<ui5-suggestion-item-custom>``` only exposes a text property for accessibility. Any extra content rendered inside (e.g. “EU (as in the example)”) is not announced by NVDA/JAWS.

This is more of a feature request than a bug, so I’m moving it to the planning board. The proposal is to introduce an ARIA property (e.g. accessibleName) so app developers can explicitly define what gets read out, instead of relying on inner HTML parsing that might include decorative elements.

Best Regards,
Stefan

**Comment by hristop** [comment]
Internal BLI created: BGSOFUIRILA-4128

**Comment by hristop** [comment]
We have raised an Inner Source item for this: SAPDESIGNSYSTEM-423

**Comment by ivoplashkov** [comment]
Hi @VidhiNPatel 

Thanks you for reaching out!

The additional text is now being announced with the text of the list item. The change has been done in the sample with the following PR - https://github.com/UI5/webcomponents/pull/13436.

Best regards,
Ivaylo
